multiversx_sdk/network_providers/user_agent.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from typing import Any, Dict
+
+from multiversx_sdk.network_providers.config import NetworkProviderConfig
+from multiversx_sdk.network_providers.constants import UNKNOWN_CLIENT_NAME
+
+logger = logging.getLogger("user_agent")
+
+
+def extend_user_agent(user_agent_prefix: str, config: NetworkProviderConfig):
+    if config.client_name is None:
+        logger.warning("Please provide the client name of the application that uses the SDK. It will be used for metrics.")
```

Comment:
Should be info, similar to JS.

_We recommend providing the name of your application when instantiating a NetworkProvider (e.g., ProxyNetworkProvider, ApiNetworkProvider). This information will be used for metrics collection and improving our services._

Reply:
Changed the message to the one you've mentioned